cmd/backup-manager/app/backup/manager.go

```diff
@@ -166,11 +159,10 @@ func (bm *Manager) performBackup(ctx context.Context, backup *v1alpha1.Backup, d
 	updatePathStatus := &controller.BackupUpdateStatus{
 		BackupPath: &backupFullPath,
 	}
-	err = bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
+	if err := bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
```

Comment:
Is it better to move this block in the `db!=nil` branch in L175? Because if `db==nil`, there is nothing to prepare.

Reply:
If db == nil, `Prepare` condition should always be false?

Reply:
What about just skip setting this condition?

Reply:
What means if `Prepare` condition is true? A prepare process is running or is finished ?

Reply:
Let's take it as the preparation process is running.